.github/workflows/r_package.yml

```diff
@@ -134,15 +134,6 @@ jobs:
             r_version: 4.3
             build_type: cran
             container: null
-          ################
-          # Other checks #
-          ################
-          - os: ubuntu-latest
-            task: r-rchk
-            compiler: gcc
-            r_version: 4.3
-            build_type: cran
-            container: 'ubuntu:22.04'
```

Comment:
This job has been silently broken for over a year.

I'm working on fixing it in #6332 . See that PR for details.

In this PR, I'm proposing removing it entirely to save some CI time and to not have to introduce complexity to workaround its issues as a part of this PR.